tests/test_edgeql_introspection.py

```diff
@@ -637,6 +637,29 @@ class TestIntrospection(tb.QueryTestCase):
             ]
         )
 
+    async def test_edgeql_introspection_function_02(self):
+        await self.assert_query_result(
+            r"""
+                WITH MODULE schema
+                SELECT `Function` {
+                    name,
+                    session_only
+                }
+                FILTER .name IN {'std::count', 'sys::advisory_lock'}
+                ORDER BY .name;
+            """,
+            [
+                {
+                    "name": "std::count",
+                    "session_only": {},
```

Comment:
Why is this an empty set as opposed to the default `False`?

Reply:
Oh, that's probably a bug then. I think I've seen this before with introspection queries specifically. I'll see if I can find more examples and put them into a separate test.